src/modal/modal-stack.ts

```diff
@@ -21,7 +21,8 @@ import {NgbModalWindow} from './modal-window';
 
 @Injectable({providedIn: 'root'})
 export class NgbModalStack {
-  private _windowAttributes = ['ariaLabelledBy', 'backdrop', 'centered', 'keyboard', 'size', 'windowClass'];
+  private _windowAttributes =
+      ['ariaLabelledBy', 'backdrop', 'centered', 'keyboard', 'size', 'windowClass', 'focusFirst'];
```

Comment:
My understanding was that we don't want to introduce this option for now. Did anything change?